src/checksums.ts

```diff
@@ -1,6 +1,10 @@
 import * as httpm from 'typed-rest-client/HttpClient'
 
-const httpc = new httpm.HttpClient('gradle/wrapper-validation-action')
+const httpc = new httpm.HttpClient(
+  'gradle/wrapper-validation-action',
+  undefined,
+  {allowRetries: true, maxRetries: 3}
```

Comment:
We can find this API from here: https://github.com/microsoft/typed-rest-client/blob/f21ab8a105826b64bf5c2df81e1f341537d273c0/test/units/httptests.ts#L422

This 3 has no special intention; I just thought that the default value 1 is too less.

Reply:
Quite a reasonable change! Thanks for this!